admin/templates/preprints/preprint.html

```diff
@@ -8,138 +8,160 @@
-            </tr>
-            <tr>
-                <td>Date Modified</td>
-                <td>{{ preprint.date_created }} UTC</td>
-            </tr>
-            <tr>
-                <td>Published</td>
-                <td>{{ preprint.is_published }}</td>
-            </tr>
-            {%  if preprint.is_published %}
-            <tr>
-                <td>Date Published</td>
-                <td>{{ preprint.date_published }} UTC</td>
-            </tr>
-            {%  endif %}
-
-            <tr>
-                <td>Creator</td>
-                <td>
-                     <a href="{{ preprint.node.creator | reverse_user }}">
-                        {{ preprint.node.creator }}
-                    </a>
-                </td>
-            </tr>
-            <tr>
-                <td>Contributors</td>
-                <td>
-                    <table class="table table-bordered table-hover">
-                    <thead>
-                        <tr>
-                            <td>User id</td>
-                            <td>Name</td>
-                            <td>Permissions</td>
-                        </tr>
-                    </thead>
-                    <tbody>
-                    {% for user in preprint.node.contributors %}
-                        <tr>
-                            <td>
-                                <a href="{{ user.id | reverse_user }}">
-                                    {{ user.id }}
+            <div class="col-md-12">
+                <table class="table table-striped">
+                <thead>
+                    <tr>
+                        <th>Field</th>
+                        <th>Value</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>GUID</td>
+                        <td>{{ serialized_preprint.id }}</td>
+                    </tr>
+                    <tr>
+                        <td>Title</td>
+                        <td>{{ serialized_preprint.node.title }}</td>
+                    </tr>
+                    <tr>
+                        <td>Node</td>
+                        <td>
+                            <a href="{{ serialized_preprint.node.id | reverse_node }}">{{ serialized_preprint.node.id }}</a>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Public</td>
+                        <td>{{ serialized_preprint.node.is_public }}</td>
+                    </tr>
+                    <tr>
+                        <td>Provider</td>
+                        <td>
+                            {{ serialized_preprint.provider.name }}
+                            <span class="provider_form_link" style="margin-left: 100px;">
+                                <a class="btn btn-link" role="button" data-toggle="collapse" href="#collapseChangeProvider">
+                                    Change prepint provider
```

Comment:
How are the permissions for this handled? There's a separate permission for editing than for viewing, yes?